.github/workflows/maven.yml

```diff
@@ -49,10 +49,9 @@ jobs:
           distribution: adopt
           java-version: ${{ matrix.java }}
       - name: Build with Maven on Java ${{ matrix.java }}
-        if: matrix.java != '11'
```

Comment:
If we don't have `if: matrix.java != '17'`, we will be running the tests twice for JDK17

Reply:
👍